samples/dnn/facial_landmarks_ie.cpp

```diff
@@ -0,0 +1,195 @@
+                                         "3: OpenCV implementation }"
+     "{ landmbackend         | 0 |   Choose one of computation backends for facedetector: "
+                                         "0: automatically (by default), "
+                                         "1: Halide language (http://halide-lang.org/), "
+                                         "2: Intel's Deep Learning Inference Engine (https://software.intel.com/openvino-toolkit), "
+                                         "3: OpenCV implementation }"
+     "{ facetarget           | 0 |   Choose one of target computation devices for facedetector: "
+                                         "0: CPU target (by default), "
+                                         "1: OpenCL, "
+                                         "2: OpenCL fp16 (half-float precision), "
+                                         "3: VPU }"
+     "{ landmtarget          | 0 |   Choose one of target computation devices for landmarkdetector: "
+                                         "0: CPU target (by default), "
+                                         "1: OpenCL, "
+                                         "2: OpenCL fp16 (half-float precision), "
+                                         "3: VPU }"
+        );
+
+    parser.about("Use this script to run classification deep learning networks using OpenCV.");
+    if (argc == 1 || parser.has("help"))
+    {
+        parser.printMessage();
+        return 0;
+    }
+
+    //Parsing input arguments
+    const std::string facexmlPath = parser.get<std::string>("facestruct");
+    const std::string facebinPath = parser.get<std::string>("faceweights");
+    const Target faceTarget = parser.get<Target>("facetarget");
+    const Backend faceBackend = parser.get<Backend>("facebackend");
+
+    const std::string landmxmlPath = parser.get<std::string>("landmstruct");
+    const std::string landmbinPath = parser.get<std::string>("landmweights");
+    const Target landmTarget = parser.get<Target>("landmtarget");
+    const Backend landmBackend = parser.get<Backend>("landmbackend");
+
+    const Scalar color = Scalar(parser.get<int>("b"), parser.get<int>("g"), parser.get<int>("r"));
+
+    //Models' definition & initialization
+    Net faceNet = readNet(facexmlPath, facebinPath);
+    faceNet.setPreferableTarget(faceTarget);
+    faceNet.setPreferableBackend(faceBackend);
+    const unsigned int faceObjectSize = 7;
+    const float faceConfThreshold = 0.7f;
+    const unsigned int facecols = 672;
+    const unsigned int facerows = 384;
+
+    Net landmNet = readNet(landmxmlPath, landmbinPath);
+    landmNet.setPreferableTarget(landmTarget);
+    landmNet.setPreferableBackend(landmBackend);
+    const unsigned int landmcols = 60;
+    const unsigned int landmrows = 60;
+    const float bb_enlarge_coefficient = 1.0f;
+
+    const std::string win1 = "FaceLandmarkDetector";
```

Comment:
win1 is not a good name in my opinion, why do you use 1?

Reply:
Because there was another window some time ago.. My mistake, thank you